tasties_app/tests/test_create_recipe.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from authConstants import VALID_USER, VALID_PASSWORD
+from tasties_app.forms import CreateRecipeForm
+from tasties_app.models import Recipe
+from django.core.exceptions import ObjectDoesNotExist
+from django.contrib.auth.models import User
+
+
+@pytest.mark.django_db
+class TestCreateRecipe:
+    def test_unauthorized_access(self, client, signed_up_credentials):
+        response = client.get("/create_recipe/")
+        assert response.status_code == 302
+        assert (
+            response.url == "/login/?next=/create_recipe/"
+        )  # verify that after login, redirect to create_recipe
+        response = client.post(
+            "/login/", data={"username": VALID_USER, "password": VALID_PASSWORD}
+        )
+        assert response.status_code == 302  # verify redirect to homepage
+        assert response.url == "/"
+
+    def test_valid_recipe(
+        self,
+        client,
+        signed_up_credentials,
+        login_to_site,
+        form_data,
+        formset_data,
+        ingredient_formset_class,
+    ):
+        response = client.get("/create_recipe/")
+        assert response.status_code == 200  # assert create_recipe page exists
+        author_id = User.objects.get(
+            username=VALID_USER
+        )  # fetch user currently logged in
+
+        recipe_form = CreateRecipeForm(
```

Comment:
Not sure this tests the view. In the view tests, you should just send a request with data.

Reply:
> Not sure this tests the view. In the view tests, you should just send a request with data.

Just made changes reflecting your suggestion.